packages/tokens/build.js

```diff
@@ -44,42 +64,80 @@ StyleDictionary.registerFormat({
 StyleDictionary.registerFormat({
   name: 'json/dtcg',
   formatter: function ({ dictionary }) {
-    const tokensObject = dictionary.allTokens.reduce(
+    // Returns proper  value for dtcg aliasing
+    const getValue = (value) => {
+      if (value.startsWith('{') && value.includes('.')) {
+        return `${value.split('.')[0]}}`
+      }
+
+      return value
+    }
+
+    // Infers type from attributes. VADS does not consistently populate the type field properly
+    const getType = (attributes) => {
+      const { category, type } = attributes
+
+      if (category.includes('color')) {
+        return 'color'
+      } else if (category === 'units') {
+        return 'dimension'
+      } else if (category === 'font' && type === 'family') {
+        return 'fontFamily'
+      } else if (category === 'font' && type === 'weight') {
+        return 'fontWeight'
+      } else if (category === 'font' && type === 'size') {
+        return 'dimension'
+      }
+
+      return ''
+    }
+
+    // Format tokens for dtcg
+    const tokens = dictionary.allTokens.reduce(
       (previousTokens, token) => ({
         ...previousTokens,
         [token.name]: {
-          $value: token.value,
-          $type: token.path[0], // path[0] is top level token type (e.g. 'color'), should meet: https://tr.designtokens.org/format/#types
+          $value: getValue(token.original.value),
+          $type: getType(token.attributes),
         },
       }),
       {},
     )
 
-    return JSON.stringify(tokensObject, undefined, 2) + `\n`
+    return JSON.stringify(sortTokensByKey(tokens), undefined, 2) + `\n`
   },
 })
 
-/** Registering a transform that strips out category from token name */
-StyleDictionary.registerTransform({
-  name: 'name/color/clean-up',
-  type: 'name',
-  transformer: (token) => {
-    return token.name.replace('SystemColor', '')
-  },
-})
+/**
+ * Transform Groups
+ */
 
 /** Registering transform group to massage output as desired for figma */
 StyleDictionary.registerTransformGroup({
   name: 'rn',
-  transforms: ['name/ti/camel', 'name/color/clean-up', 'color/hex'],
+  transforms: ['name/cti/camel', 'color/hex'],
 })
 
 /** Registering transform group to massage output as desired for figma */
 StyleDictionary.registerTransformGroup({
   name: 'figma',
-  transforms: ['name/ti/camel', 'name/color/clean-up', 'color/hex'],
+  transforms: ['name/cti/kebab', 'color/hex'],
 })
 
+/**
+ * Utils
+ */
+
+const sortTokensByKey = (obj) => {
```

Comment:
Not sure if you encountered issues with this to make it this way (as JS can be finnicky with in-place updating data), but just noting that at face value it seems weird it's instantiating a new object and repopulating it with data instead of the original object after sorting still pointing to the appropriate data for each key.

Reply:
Nvm, makes sense because JS objects aren't naturally ordered so you can't sort an existing one.